test/test_operations.py

```diff
@@ -2441,6 +2445,115 @@ def test_unsafe_buffer_pointer(self):
     self.assertGreaterEqual(buf_ptr_2, 0)
 
 
+class TestDLPack(parameterized.TestCase):
+
+  def _test_dlpack_capsule_conversion_helper(self, xla_tensor):
+    dlpt = xdlpack.to_dlpack(xla_tensor)  # dlpt1 has type PyCapsule
+    got = xdlpack.from_dlpack(dlpt)
```

Comment:
nit, `xla_tensor2` or `xla_tensor_dlpacked`. `got` is a bit confuinsg

Reply:
done